daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/TypeCalculator.scala

```diff
@@ -176,17 +176,17 @@ class KeysetValueTypeCalculatorUnordered(valueMap: HashMap[DataValuePrimitive, D
     if (valueMap.contains(x)) {
       valueMap.get(x) match {
         case Some(a) => (a, Maybe.Nope)
-        case None => (DataValue.NoValue, One(s"Value ${x} not found in keyset-value mapping"))
+        case None => (DataValue.NoValue, One(s"Value ${x} not found in enumeration dfdlx:repValues keyset-value mapping"))
       }
     } else {
-      (DataValue.NoValue, One(s"Key ${x} not found in keyset-value mapping"))
+      (DataValue.NoValue, One(s"Key ${x} not found in enumeration dfdlx:repValues keyset-value mapping"))
 
     }
   }
   override def outputTypeCalc(x: DataValuePrimitive, xType: NodeInfo.Kind): (DataValuePrimitiveNullable, Maybe[Error]) = {
     unparseMap.get(x) match {
       case Some(v) => (v, Maybe.Nope)
-      case None => (DataValue.NoValue, One(s"Value ${x} not found in keyset-value mapping"))
+      case None => (DataValue.NoValue, One(s"Value ${x} not found in enumeration dfdlx:repValues keyset-value mapping"))
```

Comment:
These all mention repValues, but don't some of these look for the actual enumeration `value` instead for unparsing?

Reply:
Fixed.